Firmware/ultralcd.cpp

```diff
@@ -3859,10 +3859,8 @@ void lcd_wizard(WizState state)
 				setTargetHotend(PLA_PREHEAT_HOTEND_TEMP);
 				lcd_display_message_fullscreen_P(_i("Now I will preheat nozzle for PLA.")); ////MSG_WIZARD_WILL_PREHEAT c=20 r=4
 				wait_preheat();
-				//unload current filament
-				unload_filament(FILAMENTCHANGE_FINALRETRACT);
```

Comment:
Why hard code it?

Reply:
Good point. Resolved in [e24c220](https://github.com/prusa3d/Prusa-Firmware/pull/4193/commits/e24c2200f12e9ead1143372a8067c77ac52d45ee)